src/NuGet.Clients/NuGet.PackageManagement.UI/Models/DetailControlModel.cs

```diff
@@ -428,6 +438,16 @@ public int PackageVulnerabilityMaxSeverity
             }
         }
 
+        public bool IsPackageVulnerable
+        {
+            get => PackageVulnerabilities?.Count > 0;
```

Comment:
Can this return null?

```suggestion
            get => PackageVulnerabilities?.Count > 0 ?? false;
```

Reply:
I don't think so because `null > 0` always returns false.

Reply:
@chgill-MSFT Thanks... the blog is a good step, but a better step would be to possibly just temporarily disable vulnerabilities as a whole for Solution PM UI until it's a complete experience. I worry about it "sometimes" showing vulnerabilities and other times not. 😕

I believe this can be done by applying a few simple changes to the flags like `IsPackageVulnerable` so it's only enabled for Solution PM UI...

If it's that trivial, what do you think about only enabling Project-level for this iteration?
**example change:**
```suggestion
            get => IsSolution && PackageVulnerabilities?.Count > 0;
```